pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -560,3 +557,27 @@ func makeISORootPath(p string) string {
 	fullpath := path.Join("/run/install/repo", p)
 	return fmt.Sprintf("file://%s", fullpath)
 }
+
+func makeKickstartSudoersPost(names []string) string {
+	if len(names) == 0 {
+		return ""
+	}
+	echoLineFmt := `echo -e "%[1]s\tALL=(ALL)\tNOPASSWD: ALL" > "/etc/sudoers.d/%[1]s"
+chmod 0440 /etc/sudoers.d/%[1]s`
+
+	slices.Sort(names)
```

Comment:
I really like this, but I don't think we can use it yet, can we? It was added in got 1.21 according to the docs and AFAIK we need to stuck with 1.19 features for now.

Reply:
Good catch.  Fixed.  I should add some unit test runs in CI for this.